web/src/show.php

```diff
@@ -122,20 +131,20 @@
         <?php endforeach ?>
     </tbody>
 </table>
+
 <table>
     <thead>
-        <tr>
-            <th>Semester</th>
-            <th>ordinal</th>
-            <th>Fakultät</th>
-            <th>Dozent</th>
-            <th>Vorlesung</th>
-            <th>Bemerkungen</th>
-        </tr>
+        <?php foreach ($lecturefields as $field => $title ): ?>
+            <th><?php out($title)?></th>
+        <?php endforeach ?>
     </thead>
     <tbody>
         <?php /** @var array $lecture */ foreach ($listLectures as $lecture): ?>
-            <tr>
+            <tr class="<?php out(chr(97 + array_search($lecture['person_id'], $similarIds)))?>" >
```

Comment:
Before calling array_search three times, we should probably introduce a variable at this point :wink: